src/layouts/ReviewRequest/components/Comments/SendCommentForm.tsx

```diff
@@ -0,0 +1,66 @@
+import { FormControl, HStack } from "@chakra-ui/react"
+import {
+  FormErrorMessage,
+  IconButton,
+  Input,
+} from "@opengovsg/design-system-react"
+import { useEffect } from "react"
+import { useForm } from "react-hook-form"
+import { BiSend } from "react-icons/bi"
+
+export interface CommentFormProps {
+  comment: string
+}
+
+interface SendContactFormProps {
+  onSubmit: (inputs: CommentFormProps) => Promise<void>
+  errorMessage: string
+}
+
+export const SendCommentForm = ({
+  onSubmit,
+  errorMessage,
+}: SendContactFormProps) => {
+  const {
+    handleSubmit,
+    register,
+    formState,
+    setError,
+  } = useForm<CommentFormProps>({
+    mode: "onBlur",
+  })
+
+  useEffect(() => {
+    if (errorMessage)
+      setError("comment", {
+        type: "server",
+        message: errorMessage,
+      })
+  }, [errorMessage, setError])
```

Comment:
could i check if the error persists between renders?

Reply:
hmm it does seem to actually - fixed this behaviour in https://github.com/isomerpages/isomercms-frontend/pull/1102/commits/0911b32d55dd730c9e67835f0af9732337234836